app/models/userole.rb

```diff
@@ -0,0 +1,3 @@
+class Userole < ApplicationRecord
```

Comment:
@dinakartejat remove this, the association is already available in dev branch. it is duplicate code.